composer.json

```
@@ -50,8 +52,10 @@
         "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
     },
     "extra": {
+        "_ezplatform_branch_for_behat_tests": "master",
```

```suggestion
        "_ezplatform_branch_for_behat_tests": "3.1",
```
We should use `master` for `master` branch, ezplatform-user 2.1 is used in Platform 3.1